doc/install.rst

```diff
@@ -33,87 +49,135 @@ scikit-image >= 0.12.3, a few examples require pandas >= 0.18.0.
     Scikit-learn now requires Python 3.5 or newer.
 
 If you already have a working installation of numpy and scipy,
-the easiest way to install scikit-learn is using ``pip`` ::
+the easiest way to install scikit-learn is using ``conda`` (see the
+`instructions for downloading conda
+<https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html>`_)::
+
+    conda install scikit-learn
 
+or ``pip``.
+In that case, in order to avoid any OS dependent issue it is strongly
```

Comment:
it's not just about OS dependent issues, it's mostly about dependency management (which is IMHO out of scope in sklearn documentation)